pkg/module/remote.go

```diff
@@ -136,6 +136,16 @@ func (r *Remote) Push(archive *comparch.ComponentArchive, overwrite bool) (ocm.C
 		return nil, err
 	}
 
+	if !overwrite {
+		versionAlreadyExists, _ := repo.ExistsComponentVersion(
```

Comment:
Any specific reasoning why to ignore the error returned by `ExistsComponentVersion` - If not I would usggest to not ignore it and have a proper error handling

Reply:
@jeremyharisch The reason I ignored it is that this function returns an error if the version does not exist.